app/page/trip-map.cjsx

```diff
@@ -0,0 +1,41 @@
+React                  = require 'react'
```

Comment:
This is implemented to the same way as the old front page was done. This implementation enables the map to be shown and go into fullscreen onClick, whereas the route page has the map under a tab. This depends on the requirements for the page